src/index.css

```diff
@@ -1,32 +1,275 @@
-body {
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab&display=swap');
+
+* {
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Montserrat', sans-serif;
 }
 
 input {
   margin: 20px;
   padding: 10px;
 }
+
+a {
+  text-decoration: none;
+  font-size: 16px;
+  color: #121212;
+}
+
+.header p {
+  color: #0290ff;
+  width: 16rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-size: 1.875rem;
+  font-weight: bold;
+  letter-spacing: 0.5px;
+}
+
+.App {
+  margin: 50px;
+  background-color: #fafafa;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 25px;
+  background-color: #fff;
+}
+
+.navlinks {
+  display: flex;
+  gap: 20px;
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+  color: #222;
+}
+
+.icon {
+  border-radius: 50%;
+  border: 1px solid #e8e8e8;
+  width: 50px;
+  height: 50px;
+  padding: 17px;
+}
+
+.navcontent {
+  display: flex;
+}
+
+.book-details {
+  display: flex;
+  background-color: #fff;
+  padding: 15px;
+  justify-content: space-between;
+  border: solid 1px #e8e8e8;
+  margin: 20px 35px 0 35px;
+}
+
+.book {
+  display: flex;
+  flex-direction: column;
+  width: 350px;
+  background-color: #fff;
+}
+
+button {
+  border: none;
+  background-color: #fff;
+}
+
+.button-contain {
+  display: flex;
+  text-decoration: none;
+  list-style: none;
+  gap: 1rem;
+}
+
+.vl {
+  border-left: 2px solid #979197;
+  height: 25px;
+}
+
+.vl2 {
+  border-left: 2px solid #979197;
+  height: 120px;
+}
+
+.button-contain button {
+  padding-right: 0.6rem;
+  color: #0290ff;
+}
+
+.book-items {
+  margin: 5px;
+}
+
+.book-items .category {
+  width: 5rem;
+  height: 1.125rem;
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #979197;
+}
+
+.book-items .title {
+  width: 12.313rem;
+  height: 1.813rem;
+  margin: 0.188rem 20.188rem 0 0;
+  font-family: 'Roboto Slab', serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+
+.book-items .author {
+  width: 6.688rem;
+  height: 1.188rem;
+  margin: 0.25rem 0 0;
+  font-family: 'Roboto Slab', serif;
```

Comment:
- [ ] Please correct the name of the font family, it is `RobotoSlab` to be similar to the Zeplin design 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/232226960-990fced8-e439-4227-ba7b-ae5ef2a7a679.png)